### ISSUE TITLE
[A11y] Clarify help text for labelling inputs, and add missing aria-label suggestion

### ISSUE THREAD

**Issue by carmacleod** [IBMa/equal-access#294]
The help text for labelling controls (pasted below) needs to list aria-label as one of the possible solutions. In addition, it could use a few more words to remind the user that their control needs to be a labelable element before they can use a `<label>` element to label it. It would be nicest if the definition of "labelable" linked to [labelable](https://html.spec.whatwg.org/multipage/forms.html#category-label) in the HTML spec.

> Form control with “button” role has no associated label”. The rule being violated gives the solutions as:
Add a for attribute to a visible label with the id of this control (e.g. <label for="controlID">);
OR put the UI element inside a visible <label> element;
OR add an aria-labelledby attribute to the control. It must point to visible text on the page that is meaningful as a label;
OR only if the design cannot have a visible label, use the title attribute to provide a label.

Please see [this slack thread](https://ibm-cloudplatform.slack.com/archives/C036P1CTN/p1610989271042200) for details.
